__mocks__/getLikesMock.js

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios';
```

Comment:
It's good that you used axios which works built-in with jest framework!

Reply:
Thanks